src/UXClient/Components/ScatterPlot/ScatterPlot.ts

```diff
@@ -212,7 +213,7 @@ class ScatterPlot extends ChartVisualizationComponent {
     }
 
     /******** DRAW UPDATE FUNCTION ********/   
-    public draw = (isFromResize = false) => {
+    public draw = (isFromResize = false, event?: any) => {
         this.activeDot = null;
```

Comment:
same here - what happens downstream since `event` is optional?

Reply:
In this case, downstream it just calls a mouse wrapper, which only uses the event to stop propagation (your comment on Legend.ts), so I think it's fine because if there's no event there's nothing to stop propagating.